WordPress/gutenberg
# Set Default state to media library for library button in image block

In Gutenberg Image Block, on click of 'Media Library' button it should open Media Library tab in Media Popup. Instead, it open last opened tab. It confusing for user when they specifically click Media Library button and end up on upload tab in media Popup.
## What?

Fixes #41473

## Testing Instructions
Please follow step by step instructions to test this PR.
1. Create new post.
2. Add Image block.
3. Click `Media Library` button.
4. Click on `Upload` Tab
5. Close Popup and click `Media Library` button.
6. Make sure it open `Media Library` tab in Dialog.


## Screenshots or screencast
